test/TEST-02-CRYPTSETUP/test.sh

```diff
@@ -83,7 +83,8 @@ cleanup_root_var() {
 }
 
 test_cleanup() {
-    cleanup_root_var
+    # ignore errors, so cleanup can continue
+    cleanup_root_var || true
```

Comment:
Hmm, isn't `|| :` a bit nicer? That's what's normally used in rpm scriplets...

Reply:
It seems it's about 50/50 which is used in the test/ dir:

systemd/test$ git grep -oE '\|\| (true|:)'
TEST-02-CRYPTSETUP/test.sh:|| true
TEST-04-JOURNAL/test-journal.sh:|| :
TEST-04-JOURNAL/test-journal.sh:|| :
TEST-15-DROPIN/test-dropin.sh:|| :
test-functions:|| true
test-functions:|| true
test-functions:|| true
test-functions:|| true
test-functions:|| true
test-functions:|| true
test-functions:|| true
test-functions:|| :
test-functions:|| :
test-functions:|| :
test-functions:|| :
test-functions:|| :
test-functions:|| :
test-functions:|| true
test-functions:|| true

I can update the PR to use || : instead if you prefer?

Reply:
oh, you already merged, well i can send a new PR converting all the || true into || : if you want

Reply:
That'd be nice. "|| :" is nicely brief.